### PR TITLE
adding specfem3d and pysep apps

### PR DESCRIPTION
**Pull request by jthet**

### CODE REVIEW

File: pysep/job.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  "parameterSet": {
+    "appArgs": [],
+    "containerArgs": [],
+    "schedulerOptions": [
+      {
+        "name": "allocation",
+        "description": "my allocation",
+        "include": true,
+        "arg": "-A TACC-ACI-CIC"
+      }
```
Comment (nathandf) [inline pysep/job.json:24]: Set "arg" prop to be empty string
Also maybe, we should git rid of it? Up to you

File: pysep/app.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{
+  "id": "pysep",
+  "owner": "${apiUserId}",
+  "version": "1.0",
+  "description": "Containerized pysep jupyter nb",
+  "jobType": "BATCH",
+  "runtime": "SINGULARITY",
+  "runtimeOptions": ["SINGULARITY_RUN"],
+  "containerImage": "docker://jthet/pysep:latest",
```
Comment (nathandf) [inline pysep/app.json:9]: I think we should change this to have the value of the official scoped docker hub repo. If one doesn't exist, let's get with Ian and see what we can do. If that's not feasible, then leave as is.

**Review by nathandf — CHANGES_REQUESTED** [review]
Regarding the review requests I left about the 'allocation ' scheduler option and container image prop, duplicate that effort for all 4 files.

**Review by jthet — COMMENTED** [review]
made changes

**Review by nathandf — APPROVED** [review]

**Pull request merged**